package.json

```diff
@@ -46,9 +47,6 @@
     "typescript": "^4.1.3",
     "validate-commit-message": "^3.0.1"
   },
-  "peerDependencies": {
```

Comment:
Removed peer dep because it is unnecessary if the lib is already in dependencies